src/DocumentDB.ChangeFeedProcessor/LeaseManagement/DocumentServiceLeaseStore.cs

```diff
@@ -19,6 +19,7 @@ internal class DocumentServiceLeaseStore : ILeaseStore
         private readonly string leaseCollectionLink;
         private readonly IRequestOptionsFactory requestOptionsFactory;
         private string lockETag;
+        private bool isPartitionedByLeasePk = false;
```

Comment:
same comment as in DocumentServiceLeaseManager -- this should abstracted into request options factory.